packages/sp-actions/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@audius/sp-actions",
+  "version": "1.0.0",
+  "description": "A utility for audius service providers to claim token rewards.",
+  "bin": {
+      "claim": "claim.js"
+  },
+  "author": "Audius",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@audius/libs": "1.2.118",
```

Comment:
Could we use 
```
  "@audius/sdk": "3.0.44",
```
instead?

That will ensure we are using the local version of sdk, which will make it easier to dev if we ever want to make changes to this package.

You can import libs from the sdk package like so:
```
import { AudiusLibs } from '@audius/sdk'
```

if there are issues with migrating it though then don't worry about it!